### ISSUE TITLE
Is it possible to burn tokens?

### ISSUE THREAD

**Issue by zachyking** [Berry-Pool/lucid#18]
```
let mintAssets = {
 [token.policyId + token.assetName]: BigInt(-1)
}

const tx = await Tx.new()
   .attachMintingPolicy({
    type: "Native",
    script: Buffer.from(policy.script.to_bytes()).toString('hex')
   })
  .mintAssets(mintAssets)
  .complete()
  ```
  
This doesn't burn the token and submit fails with `value not conserved`. Changing `BigInt(-1)` to `BigInt(1)` makes asset minted and submits successfuly. Are there any extra steps for burning?

**Comment by alessandrokonrad** [comment]
You need to explicitly say from which utxos you want to burn these assets with the collectFrom function.
I'm just wondering if we should make the coin selection aware of burned assets. Right now of course it doesn't know, since the output target amount for these assets is 0 or less than before.

**Comment by alessandrokonrad** [comment]
Okay I make it part of the coin selection by simply adding the burn value to the output target amount. So it has to find inputs that actually cointain the assets we want to burn.

**Comment by alessandrokonrad** [comment]
Should be fixed now with `0.2.1`
